source/renderer.js

```diff
@@ -37,7 +38,7 @@ ipc.callMain = (channel, data) => new Promise((resolve, reject) => {
 });
 
 ipc.answerMain = (channel, callback) => {
-	const browserWindow = electron.remote.getCurrentWindow();
+	const browserWindow = getCurrentWindow();
```

Comment:
Is there really no way to get the current window you're in?

Reply:
ipcRemote.invoke could be used. That would avoid using remote all together. I'll make another pr for that just to compare two implementations and you could pick whichever you like more 😁.

Reply:
+1 on not using remote. 
@dusansimic You could do as you suggested. Or if you want to prevent an IPC roundtrip then you could re-architect this so that the main process listens to a broad channel from all windows and then filters the answers using `event.sender.id`.